scripts/StorageCosts.py

```diff
@@ -17,14 +17,11 @@ def main():
     subregions = functions.openYaml().get('subregions_list')
     years = functions.getYears()
 
-    #Read in master list of technologies and get storage names
-    dfGeneration_raw = pd.read_csv('../dataSources/techList_AUTO_GENERATED.csv')
-    dfGeneration = dfGeneration_raw.loc[dfGeneration_raw['GENERATION'] == 'STO']
-    storages = dfGeneration['VALUE'].tolist()
-    if not storages:
-        df = pd.DataFrame(columns=['REGION','STORAGE','YEAR','VALUE'])
-        df.to_csv('../src/data/CapitalCostStorage.csv', index=False)
-        return
+    # Here, we would read in master list of technologies and get storage names, but none exist yet

```

Comment:
I agree with your logic. Lets set it up now though so that way in the future when we add in storage we dont have to hunt down this line. Like Im thinking we should read in the sto_techs dictionary from the yaml file, even though its empty right now?

Reply:
What do you think?